backend/src/controllers/reports/genericReportController.js

```diff
@@ -80,9 +80,17 @@ const getControllerFrom = () => {
       const modifiedGetReport = getReportAndSetRequestHeaders(templateType);
       controller.getReport = modifiedGetReport;
       /** Gets the query. */
-      const result = (await getDataFromModel(query, model, reply)) ?? null;
+      const result = await getDataFromModel(query, model, reply);
+      /* eslint "no-warning-comments": [1, { "terms": ["todo", "fixme"] }] */
+      // todo: Remove conditional logic, once all reports are completed
+      if ("removeme" !== result?.report) {
+        await sendToCdogs({ result, filename, templateType, request });
+      } else {
+        reply.code(418);
+        throw new Error("Report model not created");
```

Comment:
when we restore tests, I think the first test for this module would be to make sure the reply code returns as expected,

and that the new error is thrown and matches the text on L90 exactly